package.json

```diff
@@ -3,26 +3,28 @@
   "version": "0.14.0",
```

Comment:
Do we want to update the package name to use the @libp2p namespace?

Reply:
Maybe use the @chainsafe namespace same as noise?

Reply:
I really feel that for the sake of consistency having all packages under the @libp2p namespace has way more positives. The ethos of libp2p is composability. So a LEGO piece being in a foreign namespace, makes a dis-service to that goal.

I would much prefer to change `libp2-gossipsub` package name to `@libp2p/gossipsub`

Reply:
Per 2022-05-10 verbal conversation during JS triage: it was agreed to keep using the chainsafe namespace.  @wemeetagain can leave other comments.